cli/azd/pkg/rzip/rzip.go

```diff
@@ -6,55 +6,129 @@ package rzip
 import (
 	"archive/zip"
 	"io"
-	"io/fs"
+	"log"
 	"os"
 	"path/filepath"
 	"strings"
 )
 
+// CreateFromDirectory creates a zip archive from the given directory recursively,
+// that is suitable for transporting across machines.
+//
+// It resolves any symlinks it encounters.
 func CreateFromDirectory(source string, buf *os.File) error {
```

Comment:
Is this common practice of Java or similar where symlinked folders would exist in some packaged artifact?

Reply:
It's most common for python, but generally it can be used for monorepo / referencing source from other repos, #4715 was one example